rwengine/src/script/ScriptFunctions.hpp

```diff
@@ -223,7 +223,9 @@ inline void getClosestNode(const ScriptArguments& args, ScriptVec3& coord, AIGra
     
     for (const auto &node : nodes) {
         if (node->type == type) {
-            float dist = glm::distance2(coord, node->position);
+            float dist = std::abs(coord.x - node->position.x);
```

Comment:
please add a comment that this is how the game does it (manhattan-distance and has to be 3x as close on the vertical axis to be considered)

Reply:
Above the commented line or in the description?